phase1/gce/README.md

```diff
@@ -45,14 +45,14 @@ $ make deploy
 and fill complete the config wizard to deploy a kubernetes-anywhere cluster. Eventually, you will see a set of nodes when you run:
 
 ```console
-$ kubectl --kubeconfig phase1/gce/kubeconfig.json get nodes
+$ kubectl --kubeconfig phase1/gce/<cluster_name>/kubeconfig.json get nodes
```

Comment:
There's actually a Makefile target for printing the location of the kubeconfig file specifically to make this easier. I thought we had updated the documentation, but I guess I was wrong. Since you're updating it now, bonus points if you change this to:

    $ kubectl --kubeconfig $(make -s kubeconfig-path) get nodes

(or whatever variant of that actually works)